HACKING.md

```diff
@@ -1,57 +1,36 @@
 ⚠⚠⚠ THIS IS A LIVING DOCUMENT AND LIKELY TO CHANGE QUICKLY ⚠⚠⚠
 
-# Hacking on the MCD
+# Hacking on the MCO - prep
 
-1. Create a cluster using [the installer](https://github.com/openshift/installer/).  Many of the MCD developers use libvirt.  These instructions will be kept up to date generally against the leading edge of the installer.
+These instructions have been tested inside a Fedora 29 podman container (on a FSB29 host).
+It should also work to run these commands directly on a host system if you haven't yet
+containerized your workflow.
 
-1. Build a container image for the MCD and push it to a registry somewhere, e.g.
+1. Create a cluster using [the installer](https://github.com/openshift/installer/).  Many of the MCD developers use libvirt.  These instructions will be kept up to date generally against the leading edge of the installer.  Make sure you have set `KUBECONFIG` per the output of the installer.
 
-   ```sh
-   # this takes care of building the binary for you as well
-   WHAT=machine-config-daemon ./hack/build-image.sh
-   WHAT=machine-config-daemon REPO=docker.io/sdemos ./hack/push-image.sh
-   ```
-
-1. Set `KUBECONFIG` for use with `oc`
-
-   ```sh
-   export KUBECONFIG=<path to kubeconfig>
-   ```
+1. Follow the steps [in this comment](https://github.com/openshift/installer/issues/411#issuecomment-445165262) to create a proxy to your cluster.
```

Comment:
Let's just put the commands inline as well (but still keep the link) ? It's just the `setcap` and `port-forward` commands, right? We don't need the console/grafana/prometheus bits.

Reply:
I'm guessing you did `sudo setcap CAP_NET_BIND_SERVICE=+eip /usr/local/bin/oc`, right? `setcap` refuses to follow symlinks.

Reply:
Yeah.